python/paddle/fluid/tests/unittests/ir/inference/test_mkldnn_reshape_transpose_matmul_fuse_pass.py

```diff
@@ -20,10 +20,11 @@
 from auto_scan_test import PassAutoScanTest
 from program_config import ProgramConfig, TensorConfig
 
-num = 32 * 64
-
 
 class TestReshapeTransposeMatmulMkldnnFusePass(PassAutoScanTest):
```

Comment:
Could you please rename filenames and all occurrences of `MKLDNN` to `OneDNN`? I know that fuse pass name is still mkldnn, but tests can be already renamed.

Reply:
sure